projects/plugins/jetpack/_inc/lib/class.media-summary.php

```diff
@@ -262,11 +265,13 @@ public static function get( ?int $post_id, int $blog_id = 0, array $args = array
 					++$number_of_paragraphs;
 				}
 
-				$return['image']           = $extract['image'][0]['url'];
-				$return['secure']['image'] = self::ssl_img( $return['image'] );
-				++$return['count']['image'];
+				if ( isset( $extract['image'][0]['url'] ) ) {
+					$return['image']           = $extract['image'][0]['url'];
+					$return['secure']['image'] = self::ssl_img( $return['image'] );
+					++$return['count']['image'];
+				}
 
-				if ( $number_of_paragraphs <= 2 && is_countable( $extract['image'] ) && 1 === count( $extract['image'] ) ) {
+				if ( $number_of_paragraphs <= 2 && is_countable( $extract['image'] ) && 1 === count( $extract['image'] ) ) {  // @phan-suppress-current-line PhanTypePossiblyInvalidDimOffset -- We established the image offset exists with '! empty( $extract['has']['image']' earlier.
```

Comment:
This line was flagged by Phan due to the 'image' offset possibly being invalid. However as mentioned when suppressing, we established that the image offset exists earlier.
Previous change was added here, so I've opted to leave as-is: https://github.com/Automattic/jetpack/pull/31250/files#diff-8a52a367cf42b78b5fea4ec178b867663dd4979986a114b95b38fc0463fa73d0